website tutorial template + cfde portal overview pg fixes

marisalim opened the pull request:
## PR Checklist

- [x] Release label added
- [x] PR category label added
- [x] PR mergeable
- [x] Clean build logs
- [ ] Linked related issues - n/a
- [ ] Colorblindness test https://www.toptal.com/designers/colorfilter/ -n/a

## PR Description
- clarify what tutorial template is for from the website style guide.
- add back index/overview page for cfde portal use cases tutorial 

**Preview link**
https://cfde-training-and-engagement--304.com.readthedocs.build/en/304/CFDE-Internal-Training/Website-Style-Guide/0index/

## Review format
direct edits!


## Timeline
Feb 2021

marisalim committed:
clarify what tutorial template is for

marisalim committed:
add index.md back to cfde portal pages. add back index.md overview title

s-canchi commented:
The changes look good. Can users not from our team also have access to the google drive that hosts the powerpoint slide ?

josesanchez1815 commented:
@s-canchi  If the users are onboarded with the CFDE they should have access.

josesanchez1815 committed:
Add Contact Link to the Prerequisite Tab

Added a link training@cfde.atlassian.net to the prerequisite tab that says "Contact us"

josesanchez1815 committed:
Fix email account Link

Prerequisite Tab - Fix e-mail Account Link

josesanchez1815 committed:
Update E-Mail Link

s-canchi reviewed (approved):
lgtm!
- [x] spelling and grammar
- [ ] successful run of all installation and code chunks
- [ ] sufficient explanation and details for the tutorial content
- [x] code syntax and naming convention
- [x] adherence of tutorial format to style guide
- [x] functional links (inter and intra)
- [x] accessible hyperlink text

josesanchez1815 reviewed (approved):
I only added the changes to the contact us link in the prerequisite tab. It looks good to me

marisalim commented:
thanks @s-canchi and @josesanchez1815 !
